manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RedGateSoftwareLtd.RedgateInstaller
+PackageVersion: 14.0
+PackageLocale: en-US
+Publisher: Red Gate Software Ltd
+PackageName: SQL Developer Bundle

```

Comment:
This doesn't correspond to any of the ARP Entries added by the installer. Either the primary entry needs to be added under `AppsAndFeaturesEntries` or this needs to be changed